README.md

```diff
@@ -1,2 +1,99 @@
 # Shape-Based-hand-Gesture-Recognition
 Srishti 2020
+
+## Abstract
+   
+   <p align="justify">The aim of the project is to create hand bot which can replicate the movements of hands of the person who is interacting with the bot, with the help of image processing. This project is a combination of live motion detection and gesture identification.</p>
+
+![](https://github.com/Rohit-Kumar-219/Shape-Based-hand-Gesture-Recognition/blob/master/Images/Image%20identification.jpg)
+
+## Motivation 
+   <p align="justify">The motivation is to create a bot that can copy the movements of an original hand. With further motivations this bot can be upgraded to an artificial hand. This model is just to understand the coordination between the human and robotic actions. This bot aims in maximising of the patterns of the human hand, to replicate it.</p> 
+
+## Components
+   
+   - 2x Arduino 
+   - 2x NRF module
+   - 5x Servomotor
+   - Laptop (for image processing)
+   - 5x 10k Resistors
+   - 9V Batteries
+   - 2x Breadboard
+   - Jumpers
+   - Braid wire (for the stifness of the  fingures of the bot)
+   - Copper wire
+
+## Workflow
+![Workflow](https://github.com/Rohit-Kumar-219/Shape-Based-hand-Gesture-Recognition/blob/master/Images%20and%20Videos/Images/Workflow.png)
+
+## Mechanical aspect of the design
+
+### Structure
+<p align="justify">The bot has 5 servo motors connected to separate copper strings which is joined to the fingures of the bot. When the arduino of the bot recieves the signals, the corresponding set of motor is **switched on** amd hence the copper string is pulled, thus, bending the corresponding fingure. When the srint is released the braid wire in the rear part of the fingure brings the fingure into its original state.</p>
```

Comment:
As it is inside HTML tag, use `<b>...<\b>` to make text bold! `**...**` doesn't work inside HTML tags.